@navikt/core/css/list.css

```
@@ -0,0 +1,54 @@
+.navds-list ul,
+.navds-list ol {
+  padding: 0;
+}
+
+.navds-list li:not(:first-child) {
+  margin-top: var(--a-spacing-5);
+}
+
+.navds-list ul .navds-list__item {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  gap: var(--a-spacing-3);
+  padding-left: 0;
+}
+
+.navds-list ol .navds-list__item-marker--icon {
```

```suggestion
}

.navds-list ul .navds-list__item::marker {
  content: none;
}

.navds-list ol .navds-list__item-marker--icon {
```

Tror vi må sette `content: none` på default-marker. Den orginale ligger nå under SVG-en (tror jeg) som kan føre til bugs på noen devices